src/middleware/packages/ldp/services/resource/actions/put.js

```diff
@@ -113,8 +117,6 @@ module.exports = {
         if (newBlankNodes.length > 0) query += this.bindNewBlankNodes(newBlankNodes);
         query += ` }`;
 
-        console.log('query', query);
```

Comment:
OK pour supprimer ce console.log. Par contre je suggérerai d'utiliser `this.logger.debug('PUT query', query)`. Et faire de même pour POST et PATCH. Ca resterait cacher en mode normal, mais permettrait de facilement débugger les requêtes en mettant le logLevel de Moleculer à "debug". Qu'en penses-tu ?